manifests/role/hathitrust/prod_and_ruby.pp

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2020 The Regents of the University of Michigan.
+# All Rights Reserved. Licensed according to the terms of the Revised
+# BSD License. See LICENSE.txt for details.
+
+# Slurpee-3, squishee-3
```

Comment:
These are essentially HT prod app hosts, so we could use that moniker now.

Reply:
Done.